### PR TITLE
add tam7t as reviewer

### PR DESCRIPTION
**Pull request by aramase**
**What this PR does / why we need it**:
Thank you for all the contributions so far @tam7t! 

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
The criteria for membership is documented in [Secrets Store CSI Driver Membership](https://docs.google.com/document/d/1YqFVTJdxvNXhXkWogYf7AOKMnUjPFHCPuYtTgJSHohU/edit?usp=sharing)

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -5,3 +5,4 @@ approvers:
 reviewers:
 - aramase
 - ritazh
+- tam7t
```
Comment (ritazh) [inline OWNERS:8]: nit: new line

**Comment by aramase** [comment]
/hold

for reviews and discussions

**Comment by tam7t** [comment]
LGTM

**Pull request merged**